src/go.mod

```diff
@@ -1,3 +1,5 @@
 module 6.824
 
 go 1.15
+
+require github.com/stretchr/testify v1.7.0
```

Comment:
Golang doesn't recommend assertion library, the reason is that assertion tends to have less information.
https://github.com/golang/go/wiki/TableDrivenTests